decidim-comments/app/events/decidim/comments/comment_event.rb

```diff
@@ -28,8 +29,48 @@ def author_presenter
                                 end
         end
 
+        def perform_translation?
```

Comment:
These methods (`perform_translation?, content_in_same_language?, and translation_missing`) seem copypasted. Can we move this to a concern? Something like `MachineTranslatedEvent`,  `MachineTranslableEvent`, or whatever naming that you prefer.